opba-embedded-starter/src/main/resources/logback.xml

```diff
@@ -4,6 +4,12 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n</pattern>
         </encoder>
     </appender>
+    <!-- please do not remove but replace log level, thx peter -->
+    <logger name="de.adorsys.opba.tppbankingapi.mapper.generated.AccountListFacadeToRestMapper" level="debug"/>
```

Comment:
I suggest using at least WARN level here and override it if needed

Reply:
ok